examples/transfer-currency-fee.yaml

```diff
@@ -0,0 +1,76 @@
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - bob: https://bob.example/
+    alice: https://alice.example/
+    banka: https://banka.example/
+    bankb: https://bankb.example/
+
+'@id': rgh:valueflows/valueflows/master/examples/transfer-currency-fee.yaml
+'@graph':
+
+  # transfer of currency from alice to bob
+
+  - '@id': banka:e0caf1e3-4a9b-4927-a19c-3f681d234529
+    '@type': Process
+    skos:note: Facilitate moving currency between accounts
+    inScopeOf: https://banka.example/
+
+  - '@id': banka:0468e9b9-4d54-4eee-9d3f-bc76db11710a
+    '@type': EconomicEvent
+    action: transfer
+    transferOf:
+    - possession
+    - ownership
+    outputOf: banka:e0caf1e3-4a9b-4927-a19c-3f681d234529
```

Comment:
why output?

Reply:
I don't think we need to connect *transfer* flows to services. IMO `vf:transfer` flow could happen just on quantity of currency without moving it between bank accounts. I see thank `bank:transfer` fee related to moving quantity of currency between bank accounts. I think it may need equivalent of *load* and *unload* to represent picking up some amount from one account and dropping it off on another account. I see here tricky part that if we just want to split 100 credits from the account and merge it into inventoried resource 'in transfer', we don't have it 'in bank account' but 'in transit', I think we would check how banking systems represent credits already debited from one account and not credited yet on another account, which can happen when `bank:transfer` bounces.

Reply:
> IMO vf:transfer flow could happen just on quantity of currency without moving it between bank accounts.

Maybe that's the difference between possession and ownership? But in any case, I think we do need some way to know what transfer the service fee is for. Maybe there is a better way....

The trouble with it being the same as load and unload is that load and unload represent real events that can be seen separately, and accounted separately, and exchanged separately (if that would ever happen). Give and take you can't tell which one to use in an exchange because it is really just one in many ways. But as always, trade-offs.

Reply:
> Give and take you can't tell which one to use in an exchange because it is really just one in many ways.

If alice's bank account gets debited 100 units and next day bob's bank account gets credited 100 units IMO we have two separate flows. When moving units between accounts has 'bounce' alice's account gets debited 100 units first and then next day same account gets credited 100 units. Again IMO we have two separate flows not just two different perspectives on the same flow.

While this movement of units between accounts happen. Transfer of ownership can happen as one flow pre-origin, origin, mid-air, destination, post-destination. In my approach transfer of ownership happens as flow which involves quantity of a resource but neither decrements it or increments it, just affect how shares of that resource get computed at any given moment.

Reply:
>If alice's bank account gets debited 100 units and next day bob's bank account gets credited 100 units IMO we have two separate flows

which one do you use in an exchange?

Reply:
None of them, agreement may include any of them as commitment but exchange would include *own* flow with specific amount and specific resource, it would happen as flow on its own anywhere: pre-origin, orign, mid-air, destination, post-destination relative to moving units between accounts. Agents which share same bank account could just have *own* flow as part of exchange and the bank account would never get incremented or decremented. Similar with #365 shares of apples or peanuts stored together but different quantities owned by different people.

Reply:
Actually commitment at certain stage may not specify any specific resource at all, just *own*, provider * recipient, amount, resource classification (eg. MXN), which gets nicely represented with a single flow. Later details could get added about 'virtual delivery / pick-up' of that virtual resource.

Reply:
Yes, true, most commitments don't specify any specific resource.